packages/blue-sdk/src/addresses.ts

```diff
@@ -1,5 +1,10 @@
-import { type DottedKeys, entries } from "@morpho-org/morpho-ts";
-
+import {
+  type DeepPartial,
+  type DottedKeys,
+  entries,
+} from "@morpho-org/morpho-ts";
+import isPlainObject from "lodash/isPlainObject";
+import mergeWith from "lodash/mergeWith";
```

Comment:
```suggestion
import isPlainObject from "lodash.isPlainObject";
import mergeWith from "lodash.mergeWith";
```

Are individual and lighter packages, useful to minimize commonjs size (tree-shaking is only available for es modules if im not mistaken)